snuba/state/stateful_count.py

```diff
@@ -0,0 +1,59 @@
+from time import time
+from typing import Sequence, Tuple
+
+from arroyo.processing.strategies.dead_letter_queue import CountInvalidMessagePolicy
+from arroyo.processing.strategies.dead_letter_queue.policies.abstract import (
+    InvalidMessage,
+)
+from arroyo.types import TPayload
+
+from snuba.redis import redis_client
+
+
+class StatefulCountInvalidMessagePolicy(CountInvalidMessagePolicy[TPayload]):
```

Comment:
I'm not sure I get why this needs to be generic. It seems to only deal with `InvalidMessage` errors and never the message payload itself.

Reply:
@rahul-kumar-saini I think this question still stands. Why are any of the policies generic or would take a message payload?

Reply:
You're right, I didn't realize Policy doesn't need a generic payload anymore:

https://github.com/getsentry/arroyo/pull/49